examples/OCPS.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
<p>Interesting way to do this. Maybe add a comment saying the configs are found in </p><p>https://github.com/lsst/dm_config_ocps</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Also, we're going to get rid of labels in the future, I suggest using filenames here.</p>

Reply:
See below; this may mostly go away.